### PR TITLE
ISPN-14164 Fix metrics produced by the Infinispan server for Prometheus scraping

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-14164
https://issues.redhat.com/browse/JDG-5583

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]

**Comment by fax4ever** [comment]
Thanks @ryanemerson for the review

File: server/rest/src/test/java/org/infinispan/rest/metric/MetricsRestTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.infinispan.rest.helper.RestServerHelper;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+@Test(groups = "unit", testName = "rest.metric.MetricsRestTest")
+public class MetricsRestTest extends SingleCacheManagerTest {
+
+   private RestServerHelper restServer;
+   private RestClient restClient;
```
Comment (jabolina) [inline server/rest/src/test/java/org/infinispan/rest/metric/MetricsRestTest.java:18]: CI is complaining about a thread leaking, I think we need to stop these two.

Comment (fax4ever) [reply server/rest/src/test/java/org/infinispan/rest/metric/MetricsRestTest.java:18]: good catch thank you @jabolina

**Comment by fax4ever** [comment]
@jabolina thanks for the review, I fixed the test ;)

**Review by jabolina — APPROVED** [review]

**Comment by fax4ever** [comment]
Thank you, merged!

**Pull request merged**